cloud/logger/README.md

```diff
@@ -0,0 +1,40 @@
+# RainCatcher Logger Cloud
+
+RainCatcher wrapper for the Bunyan npm. This module creates an interface
```

Comment:
This module creates logging facade for node.js that is used by all Raincatcher modules. 
Users can use their own loggers by wrapping them into provided interface. 
By default module provides Bunyan logger implementation